actions/workflows/promote_package.yaml

```diff
@@ -0,0 +1,100 @@
+---
+version: '2.0'
+
+st2cd.promote_package:
+    description: Promote a package to production.
+    type: direct
+    input:
+      - package
+      - version
+      - revision
+      - os
+      - repo
+    tasks:
+
+      init:
+        action: core.local
+        input:
+          cmd: "mkdir -p /tmp/st2-up/{ubuntu/trusty,debian/wheezy,debian/jessie,el/6,el/7}"
+        publish:
+          rpm: "<% $.package %>-<% $.version %>-<% $.revision %>.x86_64.rpm"
```

Comment:
Sadly, this is not always the case.

both deb and rpm package file names could be much more complex, including epoch, different architecture and so on.

For example @dennybaa has `st2enterprise_1.3-1_all.deb` deployed.
@lakshmi-kannan wants to use `el7`/`el6` in naming for `rpm` packages which is valid case as well: `st2-1.dev-5.el7.x86_64.rpm`.